tests/contenttypes_tests/test_checks.py

```diff
@@ -216,3 +217,24 @@ class InvalidBookmark(models.Model):
                 id='fields.E001',
             )
         ])
+
+
+class ModelCheckTests(SimpleTestCase):
```

Comment:
Thanks, I don't understand what it does, but removing it somehow made my tests pass. Otherwise my check function doesn't discover model defined inside the test. Any pointers?